assets/app/scripts/controllers/overview.js

```diff
@@ -58,6 +71,99 @@ angular.module('openshiftConsole')
       console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
     };
 
+    var podRelationships = function() {
+      $scope.servicePodsNoDeployment = {};
+      $scope.solitaryPods = {};
+
+      angular.forEach($scope.pods, function(pod, name){
+        var replicationcontrollers = [];
+        var services = [];
+        angular.forEach($scope.deployments, function(deployment, depName){
+          var ls = new LabelSelector(deployment.spec.selector);
+          if (ls.matches(pod)) {
+            replicationcontrollers.push(depName);
+          }
+        });
+        angular.forEach($scope.services, function(service, svcName){
```

Comment:
build using the unfilteredServices map so we get an accurate representation of which things are not routed to by services